src/utils/common.ts

```diff
@@ -1,26 +1,34 @@
+import { isRouteErrorResponse } from '@remix-run/react';
+import { isEcomSDKError } from '~/api/types';
+
 export function getUrlOriginWithPath(url: string) {
     const { origin, pathname } = new URL(url);
     return new URL(pathname, origin).toString();
 }
 
-export function toError(value: unknown): Error {
-    if (value instanceof Error) {
-        return value;
+/**
+ * Try to find some error message in unknown value.
+ *
+ * Handles cases when value is `ErrorResponse`, `EcomSDKError`, `String` or `Object`.
```

Comment:
The only thing worth mentioning in this comment is "Handles remix ErrorResponse and wix ecom platform EcomSDKError error types".